spharpy/spherical.py

```diff
@@ -248,17 +249,24 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
-def spherical_harmonic_basis(n_max, coords):
+def spherical_harmonic_basis(
+        n_max, coordinates, normalization="n3d", channel_convention="acn",
+        phase_convention='Condon-Shortley'):
     r"""
     Calculates the complex valued spherical harmonic basis matrix.
-
-    The spherical harmonic functions are fully normalized (N3D) and include the
-    Condon-Shotley phase term :math:`(-1)^m` [#]_, [#]_.
+    See also :func:`spherical_harmonic_basis_real`.
 
     .. math::
+        Y_n^m(\theta, \phi) =  CS_m N_{nm} P_{nm}(cos(\theta)) e^{im\phi}
 
-        Y_n^m(\theta, \phi) = \sqrt{\frac{2n+1}{4\pi}
-        \frac{(n-m)!}{(n+m)!}} P_n^m(\cos \theta) e^{i m \phi}
+    where:
+    - $n$ is the degree
```

Comment:
```suggestion

    - :math:`n` is the degree
```
like this, the empty line should also fix the bulletpoints

Reply:
Yes we need :math:`` here and everywhere below. You could search for '$' everywhere and replace it...